src/main/java/seedu/address/logic/commands/addTuitionTaskCommand.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.TuitionSchedule;
+import seedu.address.model.person.TuitionTask;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+
+import java.util.List;
+
+/**
+ * Adds a tuition (task) into the schedule.
+ */
+public class AddTuitionTaskCommand extends UndoableCommand {
```

Comment:
Your file name should follow the name of this class. AddTuitionTaskCommand instead of addTuitionTaskCommand